qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -128,6 +136,14 @@ def change_virtio_media(cdrom_virtio):
         static_ip_address = utils_net.get_guest_ip_addr(session_serial, virtio_nic_mac, os_type='windows')
         if static_ip_address != params["static_ip"]:
             test.fail("Static ip is lost after upgrade driver,current ip is %s" % static_ip_address)
+        setup_dns_cmd = params["setup_dns_cmd"] % ifname
+        session_serial.cmd_status(setup_dns_cmd)
```

Comment:
Hi @heywji 

IMO, we should not set dns again, just check if previous dns setting is valid.

Thanks.

Reply:
Hi @leidwang, I tried that. If I don't set it, I will get "c:\\windows\system32\".

Reply:
This should be another problem.
Back to our test case, the checkpoint is to check if the static dns can be saved after upgrading the driver. If we set the DNS again after upgrading the driver, then our checkpoint will not be tested at all, which is meaningless.

Reply:
@leidwang You are right. I will change my code and find the root cause.

Reply:
I am fixing a similar bug, not sure if the error you hit is same with me, I will send you the patch.You can try to run your patch with my fix first.

Reply:
fixed now

Reply:
I have already solved this problem. 
There are two issues contained here, so let me explain. 
The first one about 'c:\\\windows\\\system32' comes from your patch-related problem. 
The second problem is a runtime error. The root cause of this problem comes from the fact that the developers fixed this problem in virtio-win-1.9.36-0.el9_3.iso, and I was using a lower version before. So that, it is normal that the DNS information is not found.

It works now. Thanks leidong.

Reply:
In short, I was using the wrong iso. Sorry :(